### ISSUE TITLE
`#[derive(StaticType)]` triggers the `redundant_semicolons` warning

### ISSUE THREAD

**Issue by qryxip** [Nadrieril/dhall-rust#216]
```rust
#![allow(dead_code)]

use serde_dhall::StaticType;

#[derive(StaticType)]
struct Struct {
    param: String,
}
```

```console
❯ cargo clippy
warning: unnecessary trailing semicolon
 --> src/main.rs:8:16
  |
8 |         param: String,
  |                ^^^^^^ help: remove this semicolon
  |
  = note: `#[warn(redundant_semicolons)]` on by default

warning: 1 warning emitted

    Finished dev [unoptimized + debuginfo] target(s) in 0.05s
```

```console
❯ rustc --version
rustc 1.51.0 (2fd73fabe 2021-03-23)
❯ rustc -W help | rg redundant-semicolons | head -n1
                                       redundant-semicolons  warn     detects unnecessary trailing semicolons
```

https://github.com/Nadrieril/dhall-rust/blob/846c14f92bda2fb3e68c3debf940414628013574/dhall_proc_macros/src/derive.rs#L150-L154

**Comment by Nadrieril** [comment]
Fixed in https://github.com/Nadrieril/dhall-rust/pull/215

**Comment by Nadrieril** [comment]
The fix is in `v0.10.1`
